cdc/sink/codec/json.go

```diff
@@ -38,7 +38,7 @@ const (
 	// DefaultMaxMessageBytes sets the default value for max-message-bytes
 	DefaultMaxMessageBytes int = 64 * 1024 * 1024 // 64M
```

Comment:
What about 1MB? 64MB exceeds Kafka's default `max.message.bytes` (1MB) https://kafka.apache.org/documentation/#brokerconfigs

Reply:
`1MB` is not a suitable default value, the max size of a value in tikv is 6MB, so the max size of a row event approximately equal to 12MB (with old value), the default value should make sure one row event at lastest can be sent to kafka

Reply:
The current implementation tries to send out the Kafka message with only one event, **no matter what**. The codec will print a warning if the message is larger than the configured value.